packages/console/src/components/ItemPreview/index.tsx

```diff
@@ -7,15 +7,20 @@ import * as styles from './index.module.scss';
 type Props = {
   title: ReactNode;
   subtitle?: string;
-  icon?: ReactNode;
+  icon?: ReactNode | string;
   to?: To;
   size?: 'default' | 'compact';
 };
 
 const ItemPreview = ({ title, subtitle, icon, to, size = 'default' }: Props) => {
   return (
     <div className={classNames(styles.item, styles[size])}>
-      {icon && <div className={styles.icon}>{icon}</div>}
+      {icon &&
+        (typeof icon === 'string' ? (
+          <img className={styles.icon} src={icon} />
+        ) : (
+          <div className={styles.icon}>{icon}</div>
+        ))}
```

Comment:
just curious since all AC resources are internal reference only. Should we align the icon usage here? instead of some are URL some are nodes

Reply:
No.
Connector icons are loaded from `connector.metadata.logo`.
This may be refactored in LOG-2446.